src/components/KymaModules/CommunityModulesAddModule.tsx

```diff
@@ -160,18 +152,11 @@ export default function CommunityModulesAddModule(props: any) {
       return getAvailableCommunityModules(
         notInstalledCommunityModuleTemplates,
         {} as ModuleTemplateListType,
-        moduleReleaseMetas,
       );
     } else {
       return new Map();
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [
-    notInstalledCommunityModuleTemplates,
-    installedCommunityModules,
-    moduleReleaseMetas,
-    moduleReleaseMetasLoading,
-  ]);
+  }, [notInstalledCommunityModuleTemplates, moduleReleaseMetasLoading]);
```

Comment:
If I understand correctly, we need to remove MRM completely. Do we need to wait for it to load?